.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: scala
+scala:
+  - 2.12.8
+
+os:
+  - osx
+  - linux
+
+addons:
+  homebrew:
+    packages:
+    - gmp
+    - re2
+    - re2c
+    - pkgconfig
+    - python3
+    - libtool
+    - protobuf
```

Comment:
You need to add `update: true` for Homebrew to successfully install these packages

Reply:
I'm using a newer Xcode image to fix this `osx_image: xcode10.2`, because brew update can take a very long time.